app/javascript/react/components/reviews/ReviewTile.js

```diff
@@ -2,17 +2,29 @@ import React from "react"
 import VillagerIcon from "../../../../assets/images/villager.png"
 
 const ReviewTile = props => {
-  const { title, description, rating, user } = props
+  const { title, description, rating, postedUser, deleteReview, currentUser } = props
+
+  const clickHandler = (event) => {
+    event.preventDefault()
+    deleteReview()
+  }
+
+  const deleteButton = () => {
+    if (currentUser.role === "admin"){
+      return <button className="button alert" onClick={clickHandler} >Delete</button>
+    }
+  }
 
   return(
     <div className="callout">
       <span>
-        <img className="review-avatar" src={VillagerIcon}/>
-        <span>{user}</span>
+          <img className="review-avatar" src={postedUser.profile_photo.url}/>
+          <span>{postedUser.username}</span>
       </span>
       <h3>{title}</h3>
       <p>{description}</p>
       <p>Rating: {rating}</p>
+      {deleteButton()}
```

Comment:
So we don't actually need this to be a function (I missed this earlier!). What you can do is have something like 
```
let deleteButton
if (currentUser.role === "admin") {
deleteButton = <button....
}
```

Then on line 27 it would just be `{deleteButton}`